README.md

```diff
@@ -4,7 +4,7 @@
 
 1. `nvm use`
 2. `npm install`
-3. `API_KEY=xxxxxxx PORT=4444 SPLITIO_DEBUG='on' npm start`
+3. `API_KEY=xxxxxxx SPLITIO_DEBUG='on' npm start`
```

Comment:
doesn't `SPLITIO_SERVER_PORT` apply here?

Reply:
Yes, you can use it on Node alone as well. I'll add it back.